src/hazard/sources/osc_zarr.py

```diff
@@ -149,8 +146,8 @@ def write(
         chunks: Optional[List[int]] = None,
         spatial_coords: Optional[bool] = True,
     ):
-        if self.extra_xarray_store and spatial_coords:
-            self.write_data_array(f"{path}-xarray", da)
+        if self.write_xarray_compatible_zarr and spatial_coords:
```

Comment:
Would be great to add a docstring to this method that also explains how the `write_xarray_compatible_zarr` and `spatial_coords` should be used.

Reply:
👍 Yeah will dive into this.